help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Dynamics 365] in the **[!UICONTROL Mapping ID]** field.
+
+> [!IMPORTANT]
+>
+> The value used for the Mapping ID should exactly match the name of the custom field attribute created within [!DNL Dynamics 365].
+
+An example is shown below:
+![Schedule segment export](../../assets/catalog/crm/microsoft-dynamics-365/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/crm/microsoft-dynamics-365/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/crm/microsoft-dynamics-365/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/crm/microsoft-dynamics-365/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/crm/microsoft-dynamics-365/segment.png)
+
+1. Login to the Dynamics 365 website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute that was provided in the **Mapping ID** field during the **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
+![Dynamics 365 Contacts](../../assets/catalog/crm/microsoft-dynamics-365/contacts.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+### Authentication errors encountered while pushing events to destination {#auth-errors}
+During execution the [!DNL Dynamics 365] access token might become invalid, in such scenarios you will obtain the error message below, To resolve re-authenticate and reattempt the execution.
+![Error](../../assets/catalog/crm/microsoft-dynamics-365/error-auth.png)
+
+### Unknown errors encountered while pushing events to destination {#unknown-errors}
+When checking a dataflow run, if you see the error message below, verify that the Mapping ID that you provided in [!DNL Dynamics 365] for your Platform segment is valid and exists within [!DNL Dynamics 365].
```

Comment:
Please rephrase this section similar to the example I provided for the one above.